src/utils/gaa.js

```diff
@@ -187,19 +187,35 @@ const REGWALL_HTML = `
 </div>
 `;
 
+/**
+ * GoogleUser object that Google Sign-In returns after users sign in.
+ * https://developers.google.com/identity/sign-in/web/reference#googleusergetbasicprofile
+ * @typedef {{
+ *   getAuthResponse: function(): {id_token: string},
+ *   getBasicProfile: function(): {
+ *     getName: function(): string,
+ *     getGivenName: function(): string,
+ *     getFamilyName: function(): string,
+ *     getImageUrl: function(): string,
+ *     getEmail: function(): string,
+ *   },
+ * }} GoogleSignInUserDef
+ */
+let GoogleSignInUserDef;
```

Comment:
nit: Maybe rename to GoogleUserBasicProfile (or similar) to match the name used on the Google developer site.

Reply:
I updated this to GoogleUser, to match the docs. The basic profile is something the GoogleUser contains